features/contract-state/Totals.tsx

```diff
@@ -131,20 +156,31 @@ const Totals = () => {
               of <White>synthetic tokens</White> outstanding
             </Label>
             <LinksContainer>
-              <SmallLink
-                href={getEtherscanUrl(tokenAddress)}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Etherscan
-              </SmallLink>
-              <SmallLink
-                href={getExchangeLinkToken}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {exchangeName}
-              </SmallLink>
+              {tokenAddress && (
+                <SmallLink
+                  href={getEtherscanUrl(tokenAddress)}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Etherscan
+                </SmallLink>
+              )}
+
+              {getExchangeLinkToken != "" && (
+                <SmallLink
+                  href={getExchangeLinkToken}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {exchangeName}
+                </SmallLink>
+              )}
+
+              {tokenAddress && (
```

Comment:
Also should check that `tokenAddress !== ""` to be typesafe

Reply:
given this is now passed in from the `renderComponent` with a default string I agree should be checked against an empty string. Have updated it.